test/component/ResponsiveContainer.spec.tsx

```diff
@@ -252,8 +253,8 @@ describe('<ResponsiveContainer />', () => {
     expect(container.querySelector('.recharts-responsive-container')).toHaveStyle({
       width: '100px',
       height: '100px',
-      backgroundColor: 'red',
-      color: 'red',
+      'background-color': 'rgb(255,0,0)',
+      color: 'rgb(255,0,0)',
```

Comment:
Looks like color enum values like `'red'` get coerced into the rgb format, so updated the test to reflect that. Not sure if this is problematic.

Reply:
should be fine